.github/bin/make-compilation-db.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+# Copyright 2021 The Verible Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -u
+set -e
```

Comment:
Suggestion: also `set -o pipefail` to detect errors in a pipeline:
```shell
$ false | cat ; echo $?
0
$ set -o pipefail
$ false | cat ; echo $?
1
```

Reply:
sounds good, done.